app/views/users/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<h3> Your Events </h3>
+<p>
+   <ul id = "users">
+      <% @user.created_events.each do |event|%>
```

Comment:
- [ ] This logic should be in the show action of the users controller. This was pointed out to you in the previous review.
requirements 👇 

<img width="546" alt="Screen Shot 2020-12-06 at 9 58 45 AM" src="https://user-images.githubusercontent.com/48804258/101283625-a3151280-37a9-11eb-8700-6fd13006c7ee.png">